Source/WebCore/page/DragController.cpp

```diff
@@ -225,7 +226,7 @@ inline static bool dragIsHandledByDocument(DragHandlingMethod dragHandlingMethod
     return dragHandlingMethod != DragHandlingMethod::None && dragHandlingMethod != DragHandlingMethod::PageLoad;
 }
 
-bool DragController::performDragOperation(DragData&& dragData)
+bool DragController::performDragOperation(DragData&& dragData, LocalFrame& frame)
```

Comment:
It seems like frame is always the main frame?  There should be an assertion that it is the main frame if that is the case, but I think it should instead be removed.

Reply:
This isn't the main frame whenever performDragOperation is called with a non-main frame (so after WebPageProxy::performDragOperation got a reply that it needs to look in a given remote to the main frame sub-frame.